lib/rubocop/cop/style/commented_keyword.rb

```diff
@@ -10,7 +10,7 @@ module Style
       #
       # Note that some comments
       # (`:nodoc:`, `:yields:`, `rubocop:disable` and `rubocop:todo`)
-      # and RBS::Inline annotation comments are allowed.
+      # and RBS::Inline annotation, Steep annotation (`steep:ignore`) comments are allowed.
```

Comment:
My two cents. The position of `and` might be adjustable.

```diff
-      # (`:nodoc:`, `:yields:`, `rubocop:disable` and `rubocop:todo`)
-      # and RBS::Inline annotation comments are allowed.
+      # (`:nodoc:`, `:yields:`, `rubocop:disable` and `rubocop:todo`),
+      # RBS::Inline annotation, and Steep annotation (`steep:ignore`) are allowed.
```

Reply:
@koic 

Thanks for your suggestion!

I don't have a strong opinion, but I would like to adjust the `and` to put it at the end of the parallel relationship like (`:nodoc:`, `:yields:`, `rubocop:disable` and `rubocop:todo`).

Therefore, I fixed the comment you suggested.